index.html

```diff
@@ -8,141 +8,220 @@
     <title>Portfolio</title>
     <link rel="stylesheet" href="styles.css">
 </head>
-
 <body>
+
```

Comment:
GREAT JOB, HAPPEN ❤️ 
https://www.loom.com/share/af48dcdafbad4f3ca2beb3d1f57f7353
Here is a video of me sending you a message and optional suggestions 😇

Reply:
Thank you @Amrhub